package.json

```diff
@@ -39,7 +39,7 @@
     "@babel/cli": "7.2.3",
     "@babel/core": "7.4.0",
     "@babel/runtime": "7.4.0",
-    "@testing-library/react-native": "^4.0.1",
+    "@testing-library/react-native": "^6.0.0",
```

Comment:
Why you had to bump the version?

Reply:
I'm not sure why i had to do it, but before the bump, i was getting this error when running the tests.
> You are trying to `import` a file after the Jest environment has been torn down.

I could only fix it changing the version.